samples/colorpicker/pom.xml

```diff
@@ -36,7 +36,12 @@
     <dependency>
       <groupId>org.apache.logging.log4j</groupId>
       <artifactId>log4j-slf4j-impl</artifactId>
-      <version>2.15.0</version>
+      <version>2.17.0</version>
+    </dependency>
+    <dependency>
```

Comment:
this appears to be a duplicate of the above dependency.   Could you remove one?

Reply:
removed